apps/studio/src/features/editing-experience/components/ScheduledPublishingModal/QuickSelectTimeSection.tsx

```diff
@@ -0,0 +1,68 @@
+import type { z } from "zod"
+import { HStack, Text, VStack } from "@chakra-ui/react"
+import { Badge } from "@opengovsg/design-system-react"
+import { format } from "date-fns"
+import { useFormContext } from "react-hook-form"
+
+import type { schedulePublishClientSchema } from "~/schemas/schedule"
+
+/**
+ * The times to display as 'quick-select-times' under the date/time picker
+ */
+const QUICK_SELECT_TIMES: { hours: number; minutes: number }[] = [
```

Comment:
super nit: `{ hours: number; minutes: number }` can be extracted into a type to be reused below in `QuickSelectTimeSection`